app/views/moments/show.html.erb

```diff
@@ -43,6 +43,18 @@
   </div>
 <% end %>
 
+<% if @resources.any? %>
+<div class="smallMarginTop">
+  <div class="label"><%= label_tag "What resources could help?" %> </div>
```

Comment:
There's unnecessary whitespace after the `label_tag` we should remove so that it looks like `<div class="label"><%= label_tag "What resources could help?" %></div>`.

Reply:
I also realized that this text should be in our translation files. Feel free to use Google Translate for now.

Reply:
Hi Julia!  Thank you for your comments.  I do have 2 questions, please:  
1. With regards to the white space, I'm not sure which space you are referring to? To me the code and your example look exactly the same.  Are you referring to the indent space?
<div class="label"><%= label_tag "What resources could help?" %> </div>
<div class="label"><%= label_tag "What resources could help?" %></div>
2. Do we use the Chinese(Simplified) version for Google Translate?